.github/workflows/testsuites.yml

```diff
@@ -0,0 +1,141 @@
+# SPDX-FileCopyrightText: 2024 Rivos Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: linux-riscv-ci-testsuites
+
+defaults:
+  run:
+    shell: bash -leo pipefail {0}
+
+on: pull_request
+
+concurrency:
+  group: ci-test-${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  build-kernel:
+    if:  ${{ endsWith(github.head_ref, '_test') }}
+    runs-on: self-hosted
```

Comment:
Hmm, I think this might be a bit too wide? Maybe just startsWith(github.head_ref, 'linus') and palmer's trees?

Reply:
But "_test" branches are not created automatically, are they? I'd like to limit those tests to branches we want to test explicitly, WDYT?